source/classification.py

```diff
@@ -25,6 +26,58 @@ def fit(self, train, calibrate):
             train: Table of examples used as a training set.
             calibrate: Table of examples used as a calibration set.
         """
+        self.mf_idx = None
+        for i,attr in enumerate(train.domain.attributes):
+            if attr.name == self.mondrian_feature:
+                self.mf_idx = i
+                break
+        assert self.mf_idx is not None
+        super().fit(train, calibrate)
+
+    def p_values(self, example):
+        """Compute feature-conditional (Race) p-values for every possible class.
```

Comment:
This might be my confusion but class is our binary cancer or no cancer while the label would be the race. Am I correct there? So for our purposes we would always want to do mondrain=true to show how predictions get worst for underrepresented classes?

Reply:
My understanding is 'class' and 'label' are synonymous. In both cases, they refer to out target variable that you mention. The Vazqueq-Facelli review [1] indicates in Section 2.2 that MCP is more general and that Label-Conditional MCP is a special case that just applies the mondrian approach to the target variable. The reference for this last sentence in Section 2.2 points to [2], which--at a glance--I am seeing Section 3.2.2.1 appear to indicate some support for this notion.

1. Vazquez J, Facelli JC. Conformal Prediction in Clinical Medical Sciences. J Healthc Inform Res. 2022 Jan 28;6(3):241-252. doi: 10.1007/s41666-021-00113-8. PMID: 35898853; PMCID: PMC9309105.

<https://www.researchgate.net/publication/358194800_Conformal_Prediction_in_Clinical_Medical_Sciences>

2. Alnemer, Loai & Rajab, Lama & Aljarah, Ibrahim. (2016). Conformal Prediction Technique to Predict Breast Cancer Survivability. International journal of advanced science and technology. 96. 1-10. 10.14257/ijast.2016.96.01.

https://www.researchgate.net/publication/311323289_Conformal_Prediction_Technique_to_Predict_Breast_Cancer_Survivability